massa-execution-worker/src/context.rs

```diff
@@ -411,7 +411,7 @@ impl ExecutionContext {
             data.push(1u8);
         }
         // hash the seed to get a unique address
-        let address = Address(massa_hash::Hash::compute_from(&data));
+        let address = Address::User(UserAddress(massa_hash::Hash::compute_from(&data)));
```

Comment:
This should be an SC address here. I saw that in Address you used the same type for `UserAddress` and `SCAddress`. It's ok to define the structure SCAddress in the second time but we should open an issue to not forget before merging this but in this line we could use the variant SCAddress to avoid having a wrong line of code merged.

Reply:
I misunderstood your request. Now has `Address::SC(SCAddress(...))`